### PR TITLE
Update ubuntu 18.04 entrypoint

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Update ubuntu 18.04 entrypoint

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/preview/ubuntu18.04/docker/Dockerfile [hunk 1]
```diff
@@ -74,3 +74,4 @@ LABEL maintainer="PowerShell Team <powershellteam@hotmail.com>" \
 # Use PowerShell as the default shell
 # Use array to avoid Docker prepending /bin/sh -c
 CMD [ "pwsh-preview" ]
+
```
Comment (RDIL) [inline release/preview/ubuntu18.04/docker/Dockerfile]: Also unneeded extra space here

File: release/preview/ubuntu18.04/docker/pwsh-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
```
Comment (RDIL) [inline release/preview/ubuntu18.04/docker/pwsh-entrypoint.sh]: It doesn't look like this file is used?

Comment (TravisEz13) [reply release/preview/ubuntu18.04/docker/pwsh-entrypoint.sh]: This causes the script to quite if there is an error

Comment (RDIL) [reply release/preview/ubuntu18.04/docker/pwsh-entrypoint.sh]: Right, but I'm saying the entire shell file

Comment (TravisEz13) [reply release/preview/ubuntu18.04/docker/pwsh-entrypoint.sh]: Fixed

**Comment by RDIL** [comment]
Closing this as the proposal hasn't advanced and further work most likely won't happen for some time.

**Pull request closed without merging**